packages/vscode-extension/src/debugging/RadonCDPProxyDelegate.ts

```diff
@@ -213,30 +213,40 @@ export class RadonCDPProxyDelegate implements CDPProxyDelegate {
       .catch(_.noop);
   }
 
-  private handleScriptParsed(command: IProtocolCommand): IProtocolCommand {
+  private async getSourceMapData(sourceMapURL: string) {
+    if (sourceMapURL.startsWith("data:")) {
+      const base64Data = sourceMapURL.split(",")[1];
+      const decodedData = Buffer.from(base64Data, "base64").toString("utf-8");
+      const sourceMapData = JSON.parse(decodedData);
+      return sourceMapData;
+    }
+
+    if (sourceMapURL.startsWith("http")) {
+      const data = await fetch(sourceMapURL);
+      return data.json();
```

Comment:
```suggestion
      return await data.json();
```

I know this change does not do anything and it is a matter of opinion if it is even good, but in a similar situation some time ago we agreed to indicate that the method is async with explicit await.

Reply:
Didn't event notice this is async, honestly.
Sure, let's do it.